video/out/vo_gpu_next.c

```diff
@@ -1238,7 +1238,7 @@ static int control(struct vo *vo, uint32_t request, void *data)
     case VOCTRL_PAUSE:
         if (p->is_interpolated)
             vo->want_redraw = true;
-        return VO_TRUE;
+        break;
```

Comment:
Can you add at least *some* commit message explaining this change? What bug does it fix? Why here and not in `vo_gpu`?

Reply:
But `vo_gpu` also swallows up the `VOCTRL_PAUSE` in the exact same place:

https://github.com/mpv-player/mpv/blob/master/video/out/vo_gpu.c#L216